src/routes/admin/profiles/editProfile.ts

```diff
@@ -108,6 +115,10 @@ function getUpdatedData (newConfig: any, oldConfig: AMTConfiguration): AMTConfig
     amtConfig.generateRandomPassword = newConfig.amtPassword == null ? oldConfig.generateRandomPassword : null
     amtConfig.passwordLength = newConfig.amtPassword == null ? oldConfig.passwordLength : null
   }
+  return amtConfig
+}
+
+const handleGenerateRandomMEBxPassword = (amtConfig: AMTConfiguration, newConfig: AMTConfiguration, oldConfig: AMTConfiguration): AMTConfiguration => {
```

Comment:
do we have tests thats go along with these?

Reply:
Didn't any unit tests for REST API methods as we automated tests using postman.